blockchain/src/blockchain/verify.rs

```diff
@@ -5,6 +5,7 @@ use nimiq_database::{
     traits::{ReadTransaction, WriteTransaction},
     TransactionProxy as DBTransaction, WriteTransactionProxy,
 };
+use nimiq_hash::Hash;
```

Comment:
Mmm, why do we need it?

Reply:
This is needed because we hash a transaction on the verify function in line 106. 
However, you made me notice that we were hashing the executed transaction and checking if it has been already checked in the mempool. Since the mempool only knows raw_tx_hash it would always be false and we would always re-verify every transaction. 

This was a nice catch.